pkg/vendir/fetch/cache/resources/amount.go

```diff
@@ -0,0 +1,302 @@
+// Copyright 2022 VMware, Inc.
```

Comment:
what does it mean to stick our copyright above something that has a different copyright?

Reply:
we do add our copyright to all the files with cod in the project and we have been told that this is what we should be doing